File: 前端代码/score-web/src/api/quantitative/types/quantitative.ts
export interface QuantitativeData {
    type?: string
    qid: number
    qname: string
    usualScore: number
    skillScore: number
    disScore: number
    remark: string
}

export interface QuantitativeRequestData {
    /** 当前页码 */
    currentPage?: number
    /** 查询条数 */
    size?: number
    keyword?: string
}

export type QuantitativeResponseData = ApiResponseData<{
    records: QuantitativeData[]
    total: number
}>

File: 后端代码/score-system/src/main/java/com/gxnzd/scoresystem/entity/Quantitative.java
package com.gxnzd.scoresystem.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@TableName("quantitative")
public class Quantitative {

    @TableId(type = IdType.AUTO)  //声明主键，类型自增
    private Integer qId;

    private String qName;

    private Integer usualScore;

    private Integer skillScore;

    private Integer disScore;

    private String remark;

}
